mapfishapp/src/main/webapp/app/js/GEOR_Querier.js

```diff
@@ -0,0 +1,329 @@
+/*
+ * Copyright (C) Camptocamp
+ *
+ * This file is part of geOrchestra
+ *
+ * geOrchestra is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with geOrchestra.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ * Notes about the -AT-requires and -AT-include annotations below:
+ *
+ * - -AT-requires OpenLayers/Filter/Comparison.js is required because
+ *   Styler/widgets/form/ComparisonComboBox.js, which is
+ *   included by Styler/widgets/FilterPanel.js, requires it
+ * - -AT-requires OpenLayers/Filter/Spatial.js is required because
+ *   Styler/widgets/form/SpatialComboBox.js, which is
+ *   included by Styler/widgets/SpatialFilterPanel.js, requires it
+ * - -AT-include OpenLayers/Filter/Logical.js is required because
+ *   Styler/widgets/FilterBuilder.js uses OpenLayers.Filter.Logical
+ */
+
+/*
+ * @requires OpenLayers/Filter/Comparison.js
+ * @requires OpenLayers/Filter/Spatial.js
+ * @include OpenLayers/Filter/Logical.js
+ * @include OpenLayers/Format/Filter.js
+ * @include OpenLayers/Format/XML.js
+ // see https://github.com/georchestra/georchestra/issues/482 :
+ * @include OpenLayers/Format/WKT.js
+ * @include OpenLayers/Control/ModifyFeature.js
+ * @include OpenLayers/Control/DrawFeature.js
+ * @include OpenLayers/Handler/Point.js
+ * @include OpenLayers/Handler/Path.js
+ * @include OpenLayers/Handler/Polygon.js
+ * @include GeoExt/data/LayerRecord.js
+ * @include GeoExt/data/AttributeStore.js
+ * @include Styler/widgets/FilterBuilder.js
+ * @include GEOR_waiter.js
+ * @include GEOR_ows.js
+ * @include GEOR_util.js
+ */
+
+/* globals Ext, GeoExt, OpenLayers, GEOR */
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["GeoExt" is defined but never used](https://www.codacy.com/app/georchestra/georchestra/file/2338452786/issues/source?bid=3200181&fileBranchId=3209168#l49)